support-frontend/assets/pages/[countryGroupId]/components/formOnSubmit.ts

```diff
@@ -72,6 +84,11 @@ export const submitForm = async ({
 		productFields,
 	);
 
+	const similarProductsConsent = getConsentValue(
+		formData,
+		userWasShownCheckbox,
+	);
+
```

Comment:
what do you think about skip passing a boolean into the function and evaluate it right away.
```suggestion
	const similarProductsConsent = !userWasShownCheckbox && getConsentValue(formData);

```